example/docker-compose.extra.yml

```diff
@@ -62,28 +73,26 @@ services:
     networks:
       default:
         aliases:
-         - wdqs.svc
+         - ${WDQS_HOST}
     environment:
       - WIKIBASE_HOST
-      - WDQS_HOST=wdqs.svc
-      - WDQS_PORT=9999
-    expose:
-      - 9999
```

Comment:
`expose` has not functional purpose in Docker compose files outside of documentation. All ports on a docker service are by default exposed to all other services in the same network. This was different in 1.x versions of Docker and Docker compose, and likely why this setting was a hanging around.